mobile/src/modules/auth/components/passcode/EnterPasscodeModal.tsx

```diff
@@ -42,6 +43,12 @@ class EnterPasscode extends React.PureComponent<Props> {
     onCancel && onCancel();
   }
 
+  handleReset = () => {
+    const { onReset } = this.props;
+
+    onReset && onReset();
```

Comment:
onReset is not null by default. Only, if a dev passes null...I think the check could be omitted